dev-python/pipenv/pipenv-9.0.0-r1.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python Development Workflow for Humans"
+HOMEPAGE="https://github.com/pypa/pipenv https://pypi.python.org/pypi/pipenv"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+
+RDEPEND=""
+DEPEND="${RDEPEND}
+	>=dev-python/flake8-3.0.0[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	>=dev-python/pew-0.1.26[${PYTHON_USEDEP}]
+	>=dev-python/pip-9.0.1[${PYTHON_USEDEP}]
+	>dev-python/requests-2.18.0[${PYTHON_USEDEP}]
```

Comment:
Typo? Should it be `>=`?

Reply:
@titanofold , I copied this from the previous ebuild, my intent was only to address the bug mentioned, not to fix other QA issues.
However, this seems correct to me. That is at list [how upstream defines this dependency][1].

[1]: https://github.com/pypa/pipenv/blob/e8e4899a1df5f8f5cd8f4ee7f7f4574a0efacbb9/setup.py#L28